src/main/java/org/syncninja/service/ResourceMessagingService.java

```diff
@@ -0,0 +1,26 @@
+package org.syncninja.service;
+
+import org.syncninja.util.ResourceBundleEnum;
+
+import java.util.ResourceBundle;
+
+public class ResourceMessagingService {
```

Comment:
Can we make this class static or Singleton because we want to get the Messages everywhere and the getBundle method is costly to create each time an object is from it